src/email/fetchClientEmail.ts

```diff
@@ -0,0 +1,57 @@
+import _axios from 'axios';
+import https from 'https';
+import * as CryptoJS from 'crypto-js';
+
+export const fetchClientEmail = async (clientId: string) => {
+  const axios = _axios.create({
+    httpsAgent: new https.Agent({
+      rejectUnauthorized: false,
+    }),
+  });
+
+  const response = await axios({
+    method: 'GET',
+    url: `${process.env.DEW_OPENHIM_URL}authenticate/root@openhim.org`,
+  }).catch((err: Error) => console.log(err.message));
+
+  if (!response) {
+    return '';
```

Comment:
why are we returning an empty string? Shouldn't we throw ClientNotFound or something?